docs/quick-ingestion-guides/looker/overview.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Overview
+---
+# Looker & LookML Ingestion Guide: Overview
+
+## What You Will Get Out of This Guide
+
+This guide will help you set up the Looker & LookML connectors to begin ingesting metadata into DataHub.
+ 
+
+Upon completing this guide, you will have a recurring ingestion pipeline that will extract metadata from Looker & LookML and load it into DataHub. This will include to following Looker asset types:
+
+**Looker Connector Ingested Assets:**
+
+* Dashboards
```

Comment:
I think the format here could be improved

<img width="769" alt="image" src="https://github.com/datahub-project/datahub/assets/26561086/c8dcfc0b-d733-4a3b-94e0-064816af0eff">

Reply:
This should be more look like this (please refer to the [bigquery quickstart guide](https://datahubproject.io/docs/quick-ingestion-guides/bigquery/setup/#bigquery-prerequisites) : 
<img width="764" alt="image" src="https://github.com/datahub-project/datahub/assets/26561086/86091d58-b131-4f7e-9e4b-da73a7e8cc76">

Reply:
done

Reply:
I think the line height here is still too big (see the difference between here and Advanced Guides and Reference)
<img width="720" alt="image" src="https://github.com/datahub-project/datahub/assets/26561086/2af96eb2-83f6-494d-9570-a2ad88cda0e6">